home/templates/layout/test_questions.html

```diff
@@ -0,0 +1,99 @@
+{% load custom_filters %}
```

Comment:
I have feelings that these html templates are `block`, it's not layout.
Layout is something other templates extends.
Blocks are something which is other templates can include and reuse.